setup.py

```diff
@@ -183,7 +183,12 @@ def build_cmake_dependency(self, path, options, env=None):
         os.chdir(build_dir)
         config = 'Debug' if self.debug else 'Release'
         env = dict(os.environ, **(env or {}))
-        cmake = pt.join(CMAKE_BIN_DIR, 'cmake')
+        try:
+            from cmake import CMAKE_BIN_DIR
+            cmake = pt.join(CMAKE_BIN_DIR, 'cmake')
+        except ImportError:
+            cmake = shutil.which('cmake')
+
```

Comment:
I'd replace this with a `CMAKE_PATH` that can just be `'cmake'` if the import fails.